scrapeArtists/server.js

```diff
@@ -0,0 +1,86 @@
+var request = require("request");
+
+  var listOfIDs = getIDs();
+  var artistInfo = getArtistInfo(listOfIDs);
+
+
+function getArtistInfo(listOfIDs) {
+
+    arrayofObj = listOfIDs.table;
+
+    var elemNum = 0;
+    var newObj = {
+      table: []
+    };
+
+    var finalArray = arrayofObj.map(function (artistai) {
+      return artistai.artistId;
+    });
+
+    setInterval(function(){      
```

Comment:
In order to get 100% of results, we cannot query MusicBrainz faster than every second. 
Reference: https://musicbrainz.org/doc/XML_Web_Service/Rate_Limiting